### ISSUE TITLE
Webconsole in new tab

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-web-ui#842]
There is a strange behavior using the bundled web-ui: trying to open a new web console in a tab returns a 404 error. Opening the web console not in a tab works well.

![Screenshot from 2020-05-06 13-14-36](https://user-images.githubusercontent.com/4752614/81136772-a5a55480-8f9b-11ea-9b02-764478c68905.png)

![Screenshot from 2020-05-06 13-19-53](https://user-images.githubusercontent.com/4752614/81137008-557ac200-8f9c-11ea-9764-2a9f15e30511.png)

![Screenshot from 2020-05-06 13-17-54](https://user-images.githubusercontent.com/4752614/81136919-0cc30900-8f9c-11ea-9cea-110239289fd1.png)

```
2020-05-06 13:14:31 INFO web_log.py:233 127.0.0.1 [06/May/2020:03:44:31 +0000] "GET /server/1/project/3a8db2c6-c8fe-4fb3-b9ad-d8e6bd645272/nodes/1d6def16-3a2c-4465-9c5a-49d34d547bd7 HTTP/1.1" 404 172 "http://localhost:3080/static/web-ui/server/1/project/3a8db2c6-c8fe-4fb3-b9ad-d8e6bd645272" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36"
```

The strangest thing is that there is no problem opening the web console in a tab after launching the web-ui directly from its repository (master branch) with `yarn ng server`

![Screenshot from 2020-05-06 13-24-14](https://user-images.githubusercontent.com/4752614/81137155-e8b3f780-8f9c-11ea-87ae-1f3d4d81db9d.png)

```
2020-05-06 13:16:04 INFO web_log.py:233 127.0.0.1 [06/May/2020:03:46:04 +0000] "GET /v2/projects HTTP/1.1" 200 45821 "http://localhost:4200/server/1/project/3a8db2c6-c8fe-4fb3-b9ad-d8e6bd645272/nodes/1d6def16-3a2c-4465-9c5a-49d34d547bd7" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36"
2020-05-06 13:16:04 INFO web_log.py:233 127.0.0.1 [06/May/2020:03:46:04 +0000] "GET /v2/projects/3a8db2c6-c8fe-4fb3-b9ad-d8e6bd645272/nodes/1d6def16-3a2c-4465-9c5a-49d34d547bd7 HTTP/1.1" 200 1762 "http://localhost:4200/server/1/project/3a8db2c6-c8fe-4fb3-b9ad-d8e6bd645272/nodes/1d6def16-3a2c-4465-9c5a-49d34d547bd7" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36"
2020-05-06 13:16:04 INFO base_node.py:391 Connected to Telnet server
2020-05-06 13:16:04 INFO base_node.py:397 New client has connected to console WebSocket
```

**Comment by piotrpekala7** [comment]
please add /static/web-ui between port and /server

**Comment by piotrpekala7** [comment]
Does it work then?

**Comment by grossmj** [comment]
> please add /static/web-ui between port and /server

Yes, it works if I do this :+1:

**Comment by piotrpekala7** [comment]
should be fixed in https://github.com/GNS3/gns3-web-ui/commit/f1857629698a3b9b8297e2135b9b117179b5cdc2

**Comment by grossmj** [comment]
Fixed in v2020.2.0-beta.3, thanks :+1:
